return {
    testCase1 = function()
        error("hi")
    end
}